src/windows/windows_sys.rs

```diff
@@ -69,12 +70,24 @@ impl GUID {
     }
 }
 pub type HANDLE = *mut core::ffi::c_void;
+pub type HINSTANCE = *mut core::ffi::c_void;
 pub type HKEY = *mut core::ffi::c_void;
 pub const HKEY_LOCAL_MACHINE: HKEY = -2147483646i32 as _;
 pub type HMODULE = *mut core::ffi::c_void;
 pub type HRESULT = i32;
 pub type IMAGE_FILE_MACHINE = u16;
 pub const IMAGE_FILE_MACHINE_AMD64: IMAGE_FILE_MACHINE = 34404u16;
+pub const IID_IUnknown: GUID = GUID::from_u128(0x00000000_0000_0000_c000_000000000046);
+#[repr(C)]
+pub struct IUnknown_Vtbl {
+    pub QueryInterface: unsafe extern "system" fn(
+        this: *mut core::ffi::c_void,
+        iid: *const GUID,
+        interface: *mut *mut core::ffi::c_void,
+    ) -> HRESULT,
+    pub AddRef: unsafe extern "system" fn(this: *mut core::ffi::c_void) -> u32,
+    pub Release: unsafe extern "system" fn(this: *mut core::ffi::c_void) -> u32,
+}
```

Comment:
Nit: I wonder why these are added?

Reply:
The metadata used to described `HMODULE` - required by `FreeLibrary` and friends - actually depends on `HINSTANCE`. 

<img width="249" alt="image" src="https://github.com/user-attachments/assets/90f0f732-9e56-456d-beb0-6344bb953b54" />

Similarly, `CoCreateInstance` depends on `IUnknown`.

This is what I meant by the following in the PR description:

> There are a handful of additional types in the output. Those reflect a more complete representation of the requested filters and their dependencies.

The `bindgen` crate doesn't know whether you actually need these dependencies but it includes them as there are cases where they are definitely required.

Reply:
Ah, thanks for the detailed explanation!